File: 8kyu/Python/removeExclamationMarks.py
'''
Write function RemoveExclamationMarks which removes all exclamation marks from a given string.
'''
# method 1
def remove_exclamation_marks(s):
    return s.replace('!', '')

# method 2: lambda
remove_exclamation_marks = lambda s: s.replace('!', '')

File: 8kyu/JavaScript/removeExclamationMarks.js
/*
Write function RemoveExclamationMarks which removes all exclamation marks from a given string.
*/

// method 1: filter
const removeExclamationMarks = s => s.split('').filter(e => e !== '!').join('');

// method 2: replaceAll
const removeExclamationMarks = s => s.replaceAll('!', '');
